src/test/java/com/oreilly/learningsparkexamples/java/ChapterSixExampleTest.java

```diff
@@ -0,0 +1,187 @@
+package com.oreilly.learningsparkexamples.java;
+
+import static org.mockito.AdditionalMatchers.or;
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Matchers.isNull;
+
+import com.diffblue.deeptestutils.Reflector;
```

Comment:
Deeptestutils imports not required if the test which uses reflection is removed.